addon/lib/document.js

```diff
@@ -86,25 +86,57 @@ export default EmberObject.extend({
     const segments = slugWithPath.split(".");
     const slug = segments.pop();
     const doc = this.resolveDocument(segments);
-    return doc && doc.fields.find(field => field.question.slug === slug);
+    let answer = doc && doc.fields.find(field => field.question.slug === slug);
+    if (answer === undefined) {
```

Comment:
Since you're checking for object presence
```suggestion
    if (!answer) {
```

Reply:
Fixed